jekyll/_includes/head.html

```diff
@@ -1,6 +1,6 @@
 <head>
   <meta charset="utf-8">
-  <title>Joy of Coding 2024 - June 21, 2024</title>
+  <title>Joy of Coding</title>
```

Comment:
Given that we have not yet started preparing for 2025, would it not be good to just leave the information for the 2024 edition in? That way people landing on the page will know that it's over for this year. Maybe we can in a separate change add something like 'The 2024 edition was a great success! See you next year!'.

Reply:
Good point, I'll revert this.